server/SeparateStainsMacenkoPCA/SeparateStainsMacenkoPCA.xml

```diff
@@ -30,7 +30,6 @@
       <longflag>samplePercent</longflag>
       <description>Percentage of pixels to sample.  Specify either this or --sampleApproximateTotal</description>
       <constraints>
-	<minimum>0</minimum>
 	<maximum>1</maximum>
```

Comment:
Looking at this, if this is a percent, shouldn't the maximum be 100, not 1?  Otherwise, should it be called something other than percent?